services/multilang-support/Dockerfile

```diff
@@ -1,20 +1,26 @@
-FROM pytorch/pytorch:latest
+ARG PYTORCH="2.7.1"
+ARG CUDA="11.8"
+ARG CUDNN="9"
+FROM pytorch/pytorch:${PYTORCH}-cuda${CUDA}-cudnn${CUDNN}-runtime
 
 # See README.md#docker-image for visualisation of the directory structure
 WORKDIR /app
-RUN apt-get update && apt-get install python3 python3-pip git git-lfs curl -y
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    python3 python3-pip git git-lfs curl \
+ && apt-get clean && rm -rf /var/lib/apt/lists/*
 
 # Copy requirements 
 COPY services/multilang-support/requirements.txt /app/
 
 # Upgrade pip and install requirements
-RUN pip install --upgrade pip
-RUN pip install -r requirements.txt
+RUN pip install --upgrade pip \
+ && pip install --no-cache-dir -r requirements.txt 
 
 # Clone the model
-RUN git lfs install
+# RUN git lfs install
+
 # English -> French
-RUN git clone https://huggingface.co/Helsinki-NLP/opus-mt-en-fr ./models/opus-mt-en-fr
+# RUN git clone https://huggingface.co/Helsinki-NLP/opus-mt-en-fr ./models/opus-mt-en-fr
```

Comment:
If we aren't getting the model when building the docker image, and doing it dynamically below at runtime, then we might do it way too often (every imageup, including the daily one on unicorn). Concern about too much network use, or being rate limited?